frame/staking/src/mock/voter_bags.rs

```diff
@@ -31,7 +31,7 @@ pub const EXISTENTIAL_WEIGHT: u64 = 1;
 pub const CONSTANT_RATIO: f64 = 2.0000000000000000;
 
 /// Upper thresholds delimiting the bag list.
-pub const THRESHOLDS: [u64; 65] = [5, 15, 25, 50, 100, 1000, 2000, 3000]
+// pub const THRESHOLDS: [u64; 65] = [5, 15, 25, 50, 100, 1000, 2000, 3000]; // TODO: switch to this
```

Comment:
would be good if you use these values actually in the test.

Reply:
I think these values are more clear because we can easily map them to the account balances that we set in mock.

Reply:
See the values below - the first few increment by 10, which imo is easier to reason in scenarios like seeding values in a loop